### PR TITLE
Add Travis CI

### PR DESCRIPTION
**Pull request by cnorthwood**

### CODE REVIEW

**Comment by cnorthwood** [comment]
the vast majority of these changes are the result of running prettify, the actual changes of note are in the `package.json`s and the `travis.yml` file.

File: client/package.json [hunk 1]
```diff
@@ -76,8 +78,10 @@
     "@types/recompose": "^0.24.5",
     "cypress": "^3.1.4",
     "immutable": "^3.8.2",
+    "prettier": "^1.16.4",
     "prop-types": "^15.6.1",
     "react-scripts-ts": "^2.14.0",
-    "typescript": "^2.6.2"
+    "typescript": "^2.6.2",
+    "wait-on": "^3.2.0"
```
Comment (zjmiller) [inline client/package.json]: Is `wait-on` just for cypress? Might make sense to take it out.

Comment (cnorthwood) [reply client/package.json]: It is. I quite like the idea of having all the infrastructure required to run tests in place, even if the tests aren't actually run, so if we do come back to them we don't need to mess about with the surrounding stuff as much.

Comment (zjmiller) [reply client/package.json]: Sure, sounds good.

**Pull request merged**